tools/python/transit/gtfs/download_gtfs.py

```diff
@@ -17,17 +18,42 @@
 MAX_RETRIES = 2
 MAX_SLEEP_TIMEOUT_S = 30
 
+RAW_FILE_MOBILITYDB = "raw_mobilitydb.csv"
+
 URLS_FILE_TRANSITLAND = "feed_urls_transitland.txt"
-URLS_FILE_OMD = "feed_urls_openmobilitydata.txt"
+URLS_FILE_MOBILITYDB = "feed_urls_mobilitydb.txt"
+
+URL_MOBILITYDB_GTFS_SOURCE = "https://bit.ly/catalogs-csv"
 
 THREADS_COUNT = 2
 MAX_INDEX_LEN = 4
 
-HEADERS_OMD = {"Accept": "application/json"}
 
 logger = logging.getLogger(__name__)
 
 
+def download_gtfs_sources_mobilitydb(path):
+    """Downloads the csv catalogue from Data Mobility"""
+    try:
+        req = requests.get(URL_MOBILITYDB_GTFS_SOURCE)
+        url_content = req.content
+        csv_file = open(os.path.join(path, RAW_FILE_MOBILITYDB), 'wb')
+        csv_file.write(url_content)
+        csv_file.close()
+    except requests.exceptions.HTTPError as http_err:
+        logger.error(
+            f"HTTP error {http_err} downloading zip from {URL_MOBILITYDB_GTFS_SOURCE}. Update URL_MOBILITYDB_GTFS_SOURCE to a working url.")
+
+def get_gtfs_urls_mobilitydb(path):
+    """Extracts the feed urls from the downloaded csv file"""
+    download_gtfs_sources_mobilitydb(path)
+    file= open(RAW_FILE_MOBILITYDB,encoding="UTF-8")
```

Comment:
```suggestion
    file = open(RAW_FILE_MOBILITYDB, encoding="UTF-8")
```
Please use consistent code formatting everywhere.